bioconda_utils/pkg_test.py

```diff
@@ -133,4 +137,8 @@ def test_package(path,
     cmd += channel_args
     cmd += shlex.split(mulled_args)
     logger.debug('mulled-build command: %s' % cmd)
-    return utils.run(cmd)
+
+    env = {}
```

Comment:
You are overwriting here env which is passed to subprocess and this can result in mulled not being on pass, I guess.